app/controllers/users_controller.rb

```diff
@@ -0,0 +1,12 @@
+class UsersController < ApplicationController
+  def show
+  end
+
+  private
+
+  def user
+    @user ||= User.find_by(name: (params[:id]))
```

Comment:
little nitpick, but no need to wrap `params[:id]` in parenthesis here

Reply:
cool - thanks for pointing that out